CHANGELOG.md

```diff
@@ -13,6 +14,36 @@
 - [v0.5.0](#v050---2020-08-06)
 <!-- END TOC -->
 
+-------------------------------------------------
+## v0.16.0 - 2021-05-27
+
+### Release notes
+
+- Support for multiple connectors for dex and better support for OIDC groups.
+- Check out the [upgrade guide](migration/v0.15.x-v0.16.x/upgrade-apps.md) for a complete set of instructions needed to upgrade.
+
+### Added
+
+- A new helm chart `starboard-operator`, which creates `vulnerabilityreports` with information about image vulnerabilities.
+- Dashboard in Grafana showcasing image vulnerabilities.
+- Added option to enable dex integration for ops grafana
+- Added resource request/limits for ops grafana
+- Added support for admin group for harbor
+- Rook monitoring (ServiceMonitor and PrometheusRules) and dashboards.
+
+### Changed
+
+- The project now requires `helm-diff >= 3.1.2`. Remove the old one (via `rm -rf ~/.local/share/helm/plugins/helm-diff/`), before reinstalling dependencies.
```

Comment:
Nit: but maybe this should actually be in the release notes, since you actually need to take action on this.

Reply:
Very true!